docs/src/example_networks/hodgkin_huxley_example.md

```diff
@@ -0,0 +1,8 @@
+# [Hodgkin Huxley Example](@id hodgkin_huxley_example)
```

Comment:
What is this file and modifications?

Reply:
Sorry, used my old PC for this commit and it messed up some files because it wasn't up-to-date with the repository. Will fix it in the next commit.